neard/tests/rpc_nodes.rs

```diff
@@ -594,6 +594,7 @@ fn test_get_execution_outcome(is_tx_successful: bool) {
         .start();
 
         system.run().unwrap();
+        clients.into_iter().flat_map(|c| c.2.into_iter().map(|mut a| a.join())).for_each(drop);
```

Comment:
map().for_each(drop) can be replaced with simple for_each everywhere as far as I can tell